### ISSUE TITLE
On bee start Error: unknown parameters

### ISSUE THREAD

**Issue by petfold** [ethersphere/swarm-desktop#438]
When trying to start bee-desktop swarm-desktop_0.43.1 with bee 2.2.0-06a0aca7, I get:

Error: unknown parameters:
	debug-api-addr
	debug-api-enable
time="2024-10-01T14:36:16.515Z" level="error" msg="process exited with non-zero status code: 1"

**Comment by ferencsarai** [comment]
Thank you for reporting this! We need a bit more information to investigate further:

Questions:

- On which operating system did the issue occur?
- Is the bee config file in the data-dir writable for swarm-desktop?
- Was there any configuration file overwrite after installation?

**Comment by petfold** [comment]
Linux t-XPS-L521X 6.8.0-45-generic #45~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Wed Sep 11 15:25:05 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
No overwrite before I saw the error.
Found a temporary fix suggested by Nugaon by removing the second and third lines of the config file (containing the apparently missing libraries).It worked with that change.

    On Wednesday, 2 October 2024 at 12:53:27 CEST, Ferenc Sárai ***@***.***> wrote:  
 
 


Thank you for reporting this! We need a bit more information to investigate further:

Questions:
   
   - On which operating system did the issue occur?
   - Is the bee config file in the data-dir writable for swarm-desktop?
   - Was there any configuration file overwrite after installation?
